source/hardware/controllers/native_instruments_traktor_kontrol_s2_mk1.rst

```diff
@@ -0,0 +1,183 @@
+    Turning with :hwlabel:`SHIFT` adjusts the musical key.
+    Pushing with :hwlabel:`SHIFT` resets the key to the track’s default.
+Right encoder
+    Turning halves/doubles the loop size.
+    Turning with :hwlabel:`SHIFT` beatjumps by 1 beat forward/backward, or if a loop is enabled, moves the loop 1 beat forward/backward.
+    Pushing (de)activates a loop. Pushing with :hwlabel:`SHIFT` reactivates a disabled loop, or if a loop is enabled, jumps to the loop in point and stops the deck.
+:hwlabel:`LOOP IN`/:hwlabel:`LOOP OUT` buttons
+    Manually set the loop in and out points.
+    Press and hold while moving the jog wheel to adjust the loop in or out point.
+
+Top Pad Row
+~~~~~~~~~~~
+
+The top pad row has 3 different modes.
+
+Hotcue mode
+    This is the default mode when Mixxx starts.
+    The pads control hotcues 1-4.
+    Active hotcues light up blue.
+    Press an unlit button to set a new :term:`hotcue`.
+    Press a lit pad to seek to the hotcue.
+    Press a lit pad with :hwlabel:`SHIFT` to delete the hotcue.
+Intro & Outro cue mode
+    This mode is activated by pressing the :hwlabel:`RESET` button above the tempo fader.
+    Pads 1 & 2 are used for the intro start & end cues and light up green.
+    Pads 3 & 4 are used for the outro start & end cues and light up blue.
+Sampler mode
+    This mode is activated by the button under the Sampler knob in the center of the mixer.
+    Press an unlit pad to load the selected track in the library to the sampler.
+    Loaded and stopped sampler pads are lit dimm green.
+    Press a dimm green pad to play a sampler.
+    A playing sampler is lit green.
+    Press a lit pad with :hwlabel:`SHIFT` to stop a sampler, or if it is already stopped, unload the
+    sample.
+    Looping sampler pads are lit cyan.
+
+Transport Controls
+^^^^^^^^^^^^^^^^^^
+
+The transport controls work mostly as labelled on the controller:
+
+:hwlabel:`SYNC` button
+    Press to :term:`sync` :term:`tempo`.
+    Press and hold to enable sync lock.
+    Press again to disable sync lock.
+    Press with :hwlabel:`SHIFT` to enable sync lock without needing to hold.
+:hwlabel:`CUE` button
+    Behavior depends on the :ref:`cue mode set in the Mixxx preferences <interface-cue-modes>`.
+    Press with :hwlabel:`SHIFT` to seek the beginning of the track and stop.
+:hwlabel:`PLAY` button
+    Play or pause the deck. Press with :hwlabel:`SHIFT` to toggle key lock.
+:hwlabel:`TEMPO` fader
+    Adjusts the :term:`tempo`.
+:hwlabel:`RESET` button
+    Mixxx `does not yet have a very useful flux/slip
+    mode <https://bugs.launchpad.net/mixxx/+bug/1475303>`__, so instead this
+    button toggles the top pad row to the intro/outro cues. Press the button
+    when it is lit to return the top pad row to hotcue mode.
+
+    When slip mode is implemented in Mixxx in the future, this could be
+    changed so :hwlabel:`SHIFT` + :hwlabel:`RESET` button activates intro & outro mode for the
+    pads.
```

Comment:
Mixxx has a Slip mode, which of cause behaves different than any other DJ software implements it.
Please describe just what happens in Mixxx 2.3 e.g. :https://deploy-preview-412--mixxx-manual.netlify.app/chapters/appendix/mixxx_controls.html?highlight=slip#control-[ChannelN]-slip_enabled

Discussions about future features/changes shouldn't be in the manual.

Reply:
This is a direct copy from the Mk2 manual.